### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by usmansbk**
# Project 5: Views.

In this PR, I did the following:

- Create user card partial
- Render user collection
- Render views
- Add basic styling

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,7 @@
-<h1>Here is a list of users</h1>
+<div>
+<% @users.each do |user| %>
+  <%= link_to user_path(user.id) do %>
+    <%= render partial: "user", object: user %>
+  <% end %>
```
Comment (Davidosky007) [inline app/views/users/index.html.erb]: * Kindly fix your views as nothing renders on the page and when i navigate to the URLs stated in the project requirement i get an error
image
* Note this milestone should function as stated in the requirement ie ( your views should be able to work with those URLs )
image
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @usmansbk 👋, 

Good job so far! 💪 
* Well done using partials ✔️ 
* No linter errors ✔️ 
* Descriptive commit mesages ✔️ 
* Well written PR ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
* I suggest you make your views pages match the given wireframe in the project requirement [Refer here ⏩ ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design,everything%20functions%20first.)
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (usmansbk) [reply app/views/users/index.html.erb]: @Davidosky007 You mean I should create fake data? I thought you were supposed to use rails console to populate the db?

Comment (Davidosky007) [reply app/views/users/index.html.erb]: @usmansbk yeah i think in the previous milestone that was done already so now your view pages should function as the wireframes

**Review by AshakaE — APPROVED** [review]
### Hi @usmansbk  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**